src/api.jl

```diff
@@ -12,114 +12,259 @@ import .OptimizeElbo
 
 const TILE_WIDTH = 20
 const MAX_ITERS = 50
-const MIN_FLUX = 2
+const MIN_FLUX = 2.0
 
 
-function initialze_objid(
-    objid::ASCIIString, mp_all::ModelParams{Float64},
-    cat_entries::Array{Celeste.Types.CatalogEntry},
-    images::Array{Celeste.Types.Image})
-
-  s = findfirst(mp_all.objids, objid)
-
-  relevant_sources = ModelInit.get_relevant_sources(mp_all, s);
-  cat_entries_s = cat_entries[relevant_sources];
-  tiled_blob, mp = ModelInit.initialize_celeste(images, cat_entries_s,
-                                                tile_width=20,
-                                                fit_psf=true);
-  active_s = findfirst(mp.objids, objid)
-  mp.active_sources = [ active_s ]
+"""
+Query the SDSS database for all fields that overlap the given RA, Dec range.
+"""
+function query_overlapping_fields(ramin, ramax, decmin, decmax)
```

Comment:
No need to change it in this commit, but IMO it would make more sense for api.jl to be a thin wrapper around substantive functionality that's available in modules.  This function is a good example of many currently in api.jl that seem more like useful functions than APIs.